packages/color/src/Color.ts

```diff
@@ -0,0 +1,356 @@
+     * new Color('white').toRgb(); // returns { r: 1, g: 1, b: 1 }
+     */
+    toRgb(): RgbColor
+    {
+        const [r, g, b] = this._components;
+
+        return { r, g, b };
+    }
+
+    /** Convert to a CSS-style rgba string: `rgba(255,255,255,1.0)`. */
+    toRgbaString(): string
+    {
+        const [r, g, b] = this.toUint8RgbArray();
+
+        return `rgba(${r},${g},${b},${this.alpha})`;
+    }
+
+    /**
+     * Convert to an [R, G, B] array of clamped uint8 values (0 to 255).
+     * @example
+     * import { Color } from 'pixi.js';
+     * new Color('white').toUint8RgbArray(); // returns [255, 255, 255]
+     * @param {number[]|Uint8Array|Uint8ClampedArray} [out] - Output array
+     */
+    toUint8RgbArray<T extends (number[] | Uint8Array | Uint8ClampedArray) = number[]>(out?: T): T
+    {
+        const [r, g, b] = this._components;
+
+        out = out ?? [] as T;
+
+        out[0] = Math.round(r * 255);
+        out[1] = Math.round(g * 255);
+        out[2] = Math.round(b * 255);
+
+        return out;
+    }
+
+    /**
+     * Convert to an [R, G, B] array of normalized floats (numbers from 0.0 to 1.0).
+     * @example
+     * import { Color } from 'pixi.js';
+     * new Color('white').toRgbArray(); // returns [1, 1, 1]
+     * @param {number[]|Float32Array} [out] - Output array
+     */
+    toRgbArray<T extends (number[] | Float32Array) = number[]>(out?: T): T
+    {
+        out = out ?? [] as T;
+        const [r, g, b] = this._components;
+
+        out[0] = r;
+        out[1] = g;
+        out[2] = b;
+
+        return out;
+    }
+
+    /**
+     * Convert to a hexadecimal number.
+     * @example
+     * import { Color } from 'pixi.js';
+     * new Color('white').toNumber(); // returns 16777215
+     */
+    toNumber(): number
+    {
+        return this._int;
+    }
+
+    /**
+     * Convert to a hexidecimal string.
+     * @example
+     * import { Color } from 'pixi.js';
+     * new Color('white').toHex(); // returns "#ffffff"
+     */
+    toHex(): string
+    {
+        const hexString = this._int.toString(16);
+
+        return `#${'000000'.substring(0, 6 - hexString.length) + hexString}`;
+    }
+
+    /**
+     * Convert to a hexidecimal string with alpha.
+     * @example
+     * import { Color } from 'pixi.js';
+     * new Color('white').toHexa(); // returns "#ffffffff"
+     */
+    toHexa(): string
+    {
+        const alphaValue = Math.round(this._components[3] * 255);
+        const alphaString = alphaValue.toString(16);
+
+        return this.toHex() + '00'.substring(0, 2 - alphaString.length) + alphaString;
+    }
+
+    /**
+     * Set alpha, suitable for chaining.
+     * @param alpha
+     */
+    setAlpha(alpha: number): this
+    {
+        this._components[3] = alpha;
+
+        return this;
+    }
+
+    /**
+     * Rounds the specified color according to the step.
+     * @param step - Number of steps which will be used as a cap when rounding colors
```

Comment:
```suggestion
     * @param {number} step - Number of steps which will be used as a cap when rounding colors
```

Reply:
We actually don't need this. Webdoc (our doc tool) will pick up the type from the TS

Reply:
Ok, I just noticed inconsistency, sometimes its set and sometimes its not.

Reply:
Yeah, some of this might be old. Since we converted from JSDoc to webdoc awhile ago and webdoc has gotten better at auto-detecting. That said, some docs don't do a good job of picking up internal types yet (e.g., `PIXI.Color`), but are fine with basic types (e.g., `number`, `string`, `boolean`, `number[]`, etc), in these cases we help webdoc a little by documenting these types.